tests/test_syntax/inline/test_emphasis.py

```diff
@@ -72,10 +108,16 @@ def test_standalone_understores_with_newlines(self):
             '<p>foo\n_ bar _\nbaz</p>'
         )
 
-    def test_standalone_asterisks_at_end(self):
+    def test_standalone_underscore_at_begin(self):
+        self.assertMarkdownRenders(
+            '_ foo_ bar',
+            '<p>_ foo_ bar</p>'
+        )
+
+    def test_standalone_asterisk_at_end(self):
         self.assertMarkdownRenders(
-            'foo * bar *',
-            '<p>foo * bar *</p>'
+            'foo *bar *',
+            '<p>foo *bar *</p>'
```

Comment:
Note that this change ensures that the test is actually testing what it was intended to test. By making the first `*` not standalone, the only way the test will pass is if the second one is standalone.